test/compile/scale/domain.test.ts

```diff
@@ -474,86 +474,80 @@ describe('compile/scale', () => {
       });
     });
 
-    it('should ignore order: ascending', () => {
+    it('should sort the output domain if one domain is sorted with true', () => {
       const domain = mergeDomains([{
         data: 'foo',
         field: 'a',
-        sort: {field: 'b', op: 'mean', order: 'ascending'}
+        sort: true
       }, {
         data: 'foo',
-        field: 'a',
-        sort: {field: 'b', op: 'mean'}
+        field: 'b',
       }]);
 
-      assert.deepEqual<VgDomain>(domain, {
+      assert.deepEqual(domain, {
         data: 'foo',
-        field: 'a',
-        sort: {field: 'b', op: 'mean'}
+        fields: ['a', 'b'],
+        sort: true
       });
     });
 
-    it('should merge domains with the same data', () => {
+    it('should not sort if no domain is sorted', () => {
       const domain = mergeDomains([{
         data: 'foo',
         field: 'a'
       }, {
         data: 'foo',
-        field: 'b'
+        field: 'b',
       }]);
 
-      assert.deepEqual<VgDomain>(domain, {
+      assert.deepEqual(domain, {
         data: 'foo',
-        fields: ['a', 'b'],
-        sort: true
+        fields: ['a', 'b']
       });
     });
 
-    it('should maintain sort even if false', () => {
+    it('should ignore order: ascending', () => {
       const domain = mergeDomains([{
         data: 'foo',
         field: 'a',
-        sort: false
+        sort: {field: 'b', op: 'mean', order: 'ascending'}
       }, {
         data: 'foo',
-        field: 'b',
+        field: 'a',
+        sort: {field: 'b', op: 'mean'}
       }]);
 
-      assert.deepEqual(domain, {
+      assert.deepEqual<VgDomain>(domain, {
         data: 'foo',
-        fields: ['a', 'b'],
-        sort: false
+        field: 'a',
+        sort: {field: 'b', op: 'mean'}
       });
     });
 
-    it('should ignore sort false if there is another sort', () => {
+    it('should merge domains with the same data', () => {
       const domain = mergeDomains([{
         data: 'foo',
-        field: 'a',
-        sort: false
+        field: 'a'
       }, {
         data: 'foo',
-        field: 'b',
-        sort: {
-          op: 'count'
-        }
+        field: 'b'
       }]);
 
       assert.deepEqual<VgDomain>(domain, {
         data: 'foo',
-        fields: ['a', 'b'],
-        sort: {
-          op: 'count'
-        }
+        fields: ['a', 'b']
       });
     });
 
     it('should merge domains with different data', () => {
       const domain = mergeDomains([{
         data: 'foo',
-        field: 'a'
+        field: 'a',
+        sort: true
```

Comment:
why are `sort: true` ignored here?  

Perhaps, update the test description (`it`) to explain why?

Reply:
How are they ignored? The sorted domains are still sorted.